src/main/java/core/basesyntax/FigureAreaObtain.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureAreaObtain {
```

Comment:
What is the reason for splitting the Interface and Abstract class if you use the word "Figure" in the Interface name? 
It is not OOP Abstraction.